src/pytsql/grammar/helper_generate_parsers.sh

```diff
@@ -9,8 +9,8 @@ cd "$( dirname "${BASH_SOURCE[0]}" )"
 antlr4="java -jar $ANTLR4_JAR_FILEPATH"
 
 # Optionally: Download grammar
-# curl -o TSqlParser.g4 https://raw.githubusercontent.com/antlr/grammars-v4/master/sql/tsql/TSqlParser.g4
-# curl -o TSqlLexer.g4 https://raw.githubusercontent.com/antlr/grammars-v4/master/sql/tsql/TSqlLexer.g4
+curl -o TSqlParser.g4 https://raw.githubusercontent.com/antlr/grammars-v4/master/sql/tsql/TSqlParser.g4
```

Comment:
If this is optional, and previously commented, don't you want to put it into an option of the script?